docs/sdks/server-sdks/java.md

```diff
@@ -173,6 +172,8 @@ The SDK will invoke the `logBanditAction` function with an `logData` object that
 | `modelVersion` (String)                              | Unique identifier for the version (iteration) of the bandit parameters used to determine the action probability | "falcon v123"                       |
 
 The code below illustrates an example implementation of a bandit logging callback that writes to Snowflake.
+Note that this is illustrative, as writing directly to Snowflake is not a best practice for scalability, and use of
```

Comment:
@giorgiomartini0 I'm adding this in as a bandaid for now.

I like how the node example includes code examples for the console but also a bunch of pipelines ([link](https://deploy-preview-418--eppo-data-docs.netlify.app/sdks/server-sdks/node/#define-an-assignment-logger)). That is probably what all SDK documentation should aspire to for all logging sections.

![image](https://github.com/user-attachments/assets/41de1a24-9a16-401d-9c0d-4ca5f9686b72)